roles/edpm_neutron_dhcp/templates/wrappers/haproxy.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
+set -x
+{%- endif -%}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_dhcp_sidecar_haproxy_image_name }}
+NEUTRON_CONFIG_DIR={{ edpm_neutron_dhcp_agent_config_dir }}
+NEUTRON_LIB_DIR={{ edpm_neutron_dhcp_agent_lib_dir }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
+# neutron. Typically of the form (with dnsmasq as an example):
+#
+# dnsmasq --no-hosts --no-resolv --except-interface=lo \
+#   --pid-file=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/pid  \
+#   --dhcp-hostsfile=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/host ...
+NETNS=$(ip netns identify)
+NAME=neutron-haproxy-${NETNS}
+HAPROXY_CMD='$(if [ -f /usr/sbin/haproxy-systemd-wrapper ]; then echo "/usr/sbin/haproxy -Ds"; else echo "/usr/sbin/haproxy -Ws"; fi)'
```

Comment:
I don't know details but I suspect that the wrapper is always present in our images and so we could simplify this command a bit. This would also allow to remove `bash` from the final CLI below.

Also, isn't it brittle to detect a "mode of operation" by the contents of the (common?) container image? I don't think we run haproxy under systemd, no? What if our image gets the wrapper installed in it? Will we change the behavior?

(AFAIU the wrapper is removed in RPMs. From changelog in Fedora:

```
* Fri Jan 05 2018 Ryan O'Hara <rohara@redhat.com> - 1.8.3-2
 - Remove haproxy-systemd-wrapper
```

But still, this seems brittle.